### ISSUE TITLE
Profile Editor: Flow limit and move-on-if linked

### ISSUE THREAD

**Issue by shadACII** [obiwan007/despresso#205]
**Describe the bug**
In the Profile Editor the "limit Flow" Option under "Goal" and the condition under "Move on if" are linked. For example, if limit Flow is changed to 8 ml/s, then the Move on if condition is changed as well to p > 8 bar. This still appears if the condition is changed, e.g. f < 8 ml/s.

**To Reproduce**
1. Go to one recipe with an advanced Profile
2. Click on the profile under selected profile to view the profile of this recipe.
3. Click on edit in the top right corner
4. Select "Goal"
5. Scroll down to the "limit Flow" Option.
6. Change the value of "limit Flow", seither via the - or + buttons or the slider
7. See error, as move on if condition value changes as well.

**Expected behavior**
The two values should not be linked. The Flow limit should be unrelated to the "move on if" condition.

**Screenshots**

**Tablet (please complete the following information):**
 - Device: Decent Teclast Tablet (Model K811C)
 - OS: Android 8.1.0, Kernel 3.8.79
 - Version despresso 1.0.12 (152)

**Comment by shadACII** [comment]
Just checked with newest Version 1.0.13(167) bug still present

**Comment by shadACII** [comment]
Problem doesn't seem to affect imported profiles from Visualizer through despresso. Profiles from the Decent (e.g. Londonium) however seem to be affected.

**Comment by obiwan007** [comment]
Yes the exit goal is not stored and not sent to the de1 at the moment. No idea on where/how to encode the data for the de1 in the corresponding frame. Maybe some other project already did the encoding correctly.
In the UI I just store this variable now seperately.
